composer.json

```diff
@@ -27,9 +27,9 @@
         "ocramius/proxy-manager": "^1.0|^2.0",
         "doctrine/doctrine-bundle": "~1.3",
         "liip/imagine-bundle": "~1.0",
-        "oneup/flysystem-bundle": "^1.0|^2.0",
+        "oneup/flysystem-bundle": "^3.0",
         "friendsofsymfony/http-cache-bundle": "~1.2|^1.3.8",
-        "sensio/framework-extra-bundle": "~3.0",
+        "sensio/framework-extra-bundle": "^5.0",
```

Comment:
Maybe set ^5.1 for the service id name "revert" they did, might affect us if we touch any of the services.

Reply:
Done!